test/gen_casadi_test.py

```diff
@@ -327,9 +329,10 @@ def test_if_else(self):
         ref_model.parameters[0].value = 10
         ref_model.equations = [
             y1 - ca.if_else(x > 0, 1, 0) * y_max,
-            ca.if_else(x > 1, ca.vertcat(y3 - 100, y2 - y_max),
-                       ca.if_else(x > 2, ca.vertcat(y3 - 1000, y2 - y_max - 1),
-                                  ca.vertcat(y3 - 10000, y2)))]
+            ca.if_else(x > 1, ca.vertcat(y2 - y_max, y3 - 100),
+                       ca.if_else(x > 2, ca.vertcat(y2 - y_max - 1, y3 - 1000),
+                                  ca.vertcat(y2, y3 - 10000)))
+        ]
 
```

Comment:
This required a reordering the equations, the way I ordered it within the blocks is more logical, matches the source, so fixed it here to match.